### PR TITLE
ref(metrics): Rename tag-field to tags

### PR DESCRIPTION
**Pull request by TBS1996**
It's a list, so the name should reflect that.

#skip-changelog

### CODE REVIEW

File: py/CHANGELOG.md [hunk 1]
```diff
@@ -3,9 +3,12 @@
 ## Unreleased
 
 - Add `allow_negative` to `BuiltinMeasurementKey`. Filter out negative BuiltinMeasurements if `allow_negative` is false. ([#2982](https://github.com/getsentry/relay/pull/2982))
+- Add ability to block metric tags matching a glob pattern. ([#2973](https://github.com/getsentry/relay/pull/2973))
 
 ## 0.8.44
 
+- Add ability to block metrics matching a glob pattern. ([#2954](https://github.com/getsentry/relay/pull/2954))
```
Comment (TBS1996) [inline py/CHANGELOG.md:10]: so this was added together with the 'various fixes and improvements', but this is more of a new feature. Not sure how to best retroactively add it

File: relay-dynamic-config/src/metrics.rs [hunk 2]
```diff
@@ -43,7 +43,7 @@ pub struct TagBlock {
     /// Name of metric of which we want to remove certain tags.
     pub name: GlobPatterns,
```
Comment (jjbayer) [inline relay-dynamic-config/src/metrics.rs:44]: Shouldn't this also be `"names"` then?

File: py/CHANGELOG.md (hunk 1 above)
Comment (jjbayer) [reply py/CHANGELOG.md:10]: You did it right IMO. "Various fixes & improvements" seems to be a bot thing.

**Review by jjbayer — APPROVED** [review]

File: relay-dynamic-config/src/metrics.rs (hunk 2 above)
Comment (TBS1996) [reply relay-dynamic-config/src/metrics.rs:44]: atm, we think of it as it should just be a single name pattern there, it's just the type we use allows for multiple. So, depends on if we should name after what we intend to do, or name after what we are able to do

**Pull request merged**